evm_arithmetization/src/cpu/kernel/asm/main.asm

```diff
@@ -76,8 +76,15 @@ global main:
 
 global store_initial:
     // Store the initial accounts and slots for hashing later
-    %store_initial_accounts
-    %store_initial_slots
+    #[cfg(feature = eth_mainnet)]
+    {
+        %store_initial_accounts
+        %store_initial_slots
+    }
+    #[cfg(feature = cdk_erigon)]
+    {
+        %store_initial_state
```

Comment:
Not too blocking, but these macros don't read type-specific, we may want to rename them into something more obvious (either with `type1` / `type2`, `mpt` / `smt` or something else?)